packages/transcribe/src/api/utils/resizeImage.ts

```diff
@@ -0,0 +1,29 @@
+import { remove } from 'fs-extra';
+import * as sharp from 'sharp';
+
+const resizeImage = async (filePath: string, imageMaxDimension: number) => {
```

Comment:
Change to inputPath, outputPath, imageMaxDimension.

- If there's nothing to do copy file from inputPath to outputPath
- Caller manages deleted files